libraries/chain/asset_evaluator.cpp

```diff
@@ -308,7 +308,7 @@ void_result asset_update_evaluator::do_apply(const asset_update_operation& o)
    database& d = db();
 
    // If we are now disabling force settlements, cancel all open force settlement orders
-   if( o.new_options.flags & disable_force_settle && asset_to_update->can_force_settle() )
+   if( (o.new_options.flags & disable_force_settle) && asset_to_update->can_force_settle() )
```

Comment:
I think this is a consensus changing fix, although I guess it doesn't affect production chain. Please do a replay to verify.

Reply:
& has higher precedence than &&, so adding the brackets shouldn't change the result.

Reply:
Oh. That's fine then.

Reply:
Sorry about that line. I do that because it makes the intent clearer to me. I try to only do it in sections of code I am working on.

Reply:
@jmjatlanta it's good to make the code clearer. That said, it's good as long as it doesn't change logic.